app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -427,43 +427,53 @@ class AttendeeFragment : Fragment() {
                     "Please accept the terms and conditions!", Snackbar.LENGTH_LONG).show()
                 return@setOnClickListener
             }
-            val attendees = ArrayList<Attendee>()
-            if (singleTicket) {
-                val pos = ticketIdAndQty?.map { it.second }?.indexOf(1)
-                val ticket = pos?.let { it1 -> ticketIdAndQty?.get(it1)?.first?.toLong() } ?: -1
-                val attendee = Attendee(id = attendeeViewModel.getId(),
-                    firstname = firstName.text.toString(),
-                    lastname = lastName.text.toString(),
-                    city = getAttendeeField("city"),
-                    address = getAttendeeField("address"),
-                    state = getAttendeeField("state"),
-                    email = email.text.toString(),
-                    ticket = TicketId(ticket),
-                    event = eventId)
-                attendees.add(attendee)
-            } else {
-                attendees.addAll(attendeeRecyclerAdapter.attendeeList)
+
+            val builder = AlertDialog.Builder(rootView.context)
```

Comment:
Use `requireContext()` instead of `rootView.context`

Reply:
Will do soon

Reply:
@liveHarshit Done